ably/realtime_client.go

```diff
@@ -72,37 +72,42 @@ func (c *RealtimeClient) onReconnectMsg(msg *proto.ProtocolMessage) {
 		if c.Connection.id == msg.ConnectionID {
 			if msg.Error != nil {
 				// (RTN15c2)
+
+				// I'm a bit confused here as the spec says to set CONNECTED event but
```

Comment:
The events thing is being revamped for the spec-compliant 1.2 release. See #144.